endpoint.rst

```diff
@@ -546,8 +546,13 @@ off of the hostname when using the following commands with a
 `run
 <cli/singularity_run.html>`_,
 `instance
+<<<<<<< HEAD
 <cli/singularity_instance.html>`_
```

Comment:
This can be hardcoded to apptainer since the file names in the repository are not generated